### DISCUSSION TITLE
Ability to change the Color range

### DISCUSSION THREAD

**Discussion by CypherColtC** [LizardByte/community#33] [Sunshine]
On Arch Linux, the colors running moonlight appears washed out.
It looks like it has to do with "Color range: [JPEG]"

On nvenc the Color range is [JPEG], on the software encoder (libx264) the Color range is [MPEG] and looks much better.

If it's possible to modify this, could we add this as an advanced option? - Unfortunately, I do not have a list of all the possibilities for the Color ranges.

**Comment by CypherColtC** [comment]
Capture Card vs Moonlight - Arch Linux - 

**NVEnc** Color Range [JPEG]
![image](https://user-images.githubusercontent.com/105936813/172672704-c40b0bbf-7a1e-4734-9aea-8ad961dc60af.png)

**Software Encoder** Color Range [MPEG]
![image](https://user-images.githubusercontent.com/105936813/172672857-5d7def90-f391-4fa7-95ec-8525efbeca30.png)

**Comment by KuleRucket** [comment]
On nvenc (with nvfbc) I get "Info: Color range: [MPEG]" and it all looks fine.

**Reply by CypherColtC** [reply]
I'm working on figuring out why I'm getting [JPEG] instead.

**Reply by CypherColtC** [reply]
So it's in MPEG now - I reverted to my backed up Arch fresh installation on my VM, so I'm really not sure what's up with the colors now.

![image](https://user-images.githubusercontent.com/105936813/172696552-ca543202-8edf-499f-b7c0-47b2712baced.png)

**Reply by CypherColtC** [reply]
Turns out the issue is between the keyboard and the chair,  (me) -- the colors are not so bad just a tiny bit lighter, adjusting some color calibration on my pc made it much better! -- I tested again with Ubuntu, Manjaro, HoloISO and Arch, and they all give some brightness differences but it's not the worst!
